components/BookmarkButton.js

```diff
@@ -1,26 +1,27 @@
 import Image from "next/image";
 import styled from "styled-components";
-
+const Button = styled.button`
+  position: absolute;
+  justify-self: flex-end;
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+  ${({ isDarkMode }) =>
+    isDarkMode &&
+    `
+    filter: invert(1);
+  `}
+`;
 export default function BookmarkButton({
   toggleBookmark,
   medication,
   isDarkMode,
 }) {
-  const Button = styled.button`
-    position: absolute;
-    justify-self: flex-end;
-    border: none;
-    background-color: transparent;
-    cursor: pointer;
-    ${isDarkMode &&
-    `
-      filter: invert(1);
-    `}
-  `;
   return (
     <Button
       onClick={() => toggleBookmark(medication.id)}
       aria-label={medication.isBookmarked ? "remove bookmark" : "bookmark"}
+      isDarkMode={isDarkMode}
```

Comment:
You can add the type of button: `type="button"` Otherwise it would be a submit button by default.

Reply:
Thank you for your feedback. I appreciate your suggestion, but I have already added the type attribute as "button" to ensure it functions as a button rather than a submit button by default.